crates/pallet-domains/src/lib.rs

```diff
@@ -615,12 +619,53 @@ mod pallet {
 
                 // Instantiate the genesis domain
                 let domain_config = DomainConfig::from_genesis::<T>(genesis_domain, runtime_id);
-                do_instantiate_domain::<T>(
-                    domain_config,
-                    genesis_domain.owner_account_id.clone(),
-                    Zero::zero(),
+                let domain_owner = genesis_domain.owner_account_id.clone();
+                let domain_id =
+                    do_instantiate_domain::<T>(domain_config, domain_owner.clone(), Zero::zero())
+                        .expect("Genesis domain instantiation must always succeed");
+
+                // Register domain_owner as the genesis operator.
+                let operator_config = OperatorConfig {
+                    signing_key: genesis_domain.signing_key.clone(),
+                    minimum_nominator_stake: genesis_domain
+                        .minimum_nominator_stake
+                        .saturated_into(),
+                    nomination_tax: genesis_domain.nomination_tax,
+                };
+                let operator_stake = T::MinOperatorStake::get();
+                let operator_id = do_register_operator::<T>(
+                    domain_owner.clone(),
+                    domain_id,
+                    operator_stake,
+                    operator_config,
                 )
-                .expect("Genesis domain instantiation must always succeed");
+                .expect("Genesis operator registration must succeed");
+
+                // Self-Nominate the genesis operator.
+                do_nominate_operator::<T>(
```

Comment:
The function automatically adds a pending transfer for operator as a nominator as well. This is not required

Reply:
Resolved in https://github.com/subspace/subspace/pull/1656#discussion_r1259535584